agimus-demos/Makefile

```diff
@@ -0,0 +1,22 @@
+# robotpkg Makefile for:	wip/agimus-vision
+# Created:			Long Ha Thuc on Tue, 18 Aug 2020
+#
+
+ORG=			agimus
+NAME=			agimus-demos
+VERSION=		1.1.0
+DOWNLOAD_LOC=   releases/download
+MASTER_SITE_AGIMUS=     https://github.com/agimus/
+DISTNAME=		${NAME}-${VERSION}
+
+MASTER_SITES=		${MASTER_SITE_AGIMUS:=${NAME}/${DOWNLOAD_LOC}/v${VERSION}/}
+MASTER_REPOSITORY=	${MASTER_REPOSITORY_GITHUB}${ORG}/${NAME}/${DOWNLOAD_LOC}/v${VERSION}
+LICENSE=		BSD-2-Clause
+CATEGORIES=		wip
+COMMENT=		Agimus-demos
+CMAKE_ARGS+=    -DPYTHON_STANDARD_LAYOUT=ON
```

Comment:
I think I created this `PYTHON_STANDARD_LAYOUT` in the jrl-cmakemodule, but it doesn't look like agimus-sot is using that module. Why do you need this arg ?